### ISSUE TITLE
Sprint 31 tier 2 helpdesk support

### ISSUE THREAD

**Issue by acrollet** [department-of-veterans-affairs/va.gov-cms#5146]
Sprint 31 tier 2 helpdesk support

Tier 2 expectations here: https://docs.google.com/document/d/15oe0wtGI_MdaScYpjJWOgPjBqQWUFURc19HheWswEe4/edit

### Table to help track issues and approximate time spent
| Date | Issue | Resolution | Time Spent | Also Involved |
|------|------|------------|------------|---------------|
|04/26| [Broken Link](https://dsva.slack.com/archives/CT4GZBM8F/p1619448458031400) | Naming issues -- two cities with the same name | 30m | Kevin Walsh, Steve Wirt |
|04/26| [Broken Link](https://dsva.slack.com/archives/CT4GZBM8F/p1619457935053600) | Possible transient glitch -- linked page actually present and available | 15m | Kevin Walsh, Steve Wirt |
|04/26| [Broken Link](https://dsva.slack.com/archives/C0MQ281DJ/p1619459553305200) | Identified by Vanessa -- another possible glitch | 15m | |
|04/26| [Broken Link](https://dsva.slack.com/archives/C0MQ281DJ/p1619464438308800) | Mis-entered link. | 15m |  Kevin Walsh, Steve Wirt |
|04/26| [Broken Link](https://dsva.slack.com/archives/C0MQ281DJ/p1619464438308800) | Mis-entered link (same one from earlier), camouflaged by a working link that was right next to it, complicated by a confusing configuration issue with Linkit. | 40m | Kevin Walsh, Steve Wirt |
|04/26| [Broken Link](https://dsva.slack.com/archives/C0MQ281DJ/p1619467605316200) | Re-running the content deploy seemed to fix it. | 15m | |
|04/27| [Broken Links](https://dsva.slack.com/archives/CT4GZBM8F/p1619532510102900) | Vanessa's reaching out to Stan to see if this is just because of a mass-publishing action to resolve other issues or for some other reason.  A lot of false positives.  Appears to be the result of a recent PR. | 60m |  Kevin Walsh, Steve Wirt, Neil Hastings |
|04/27| [Button Style Mismatch](https://dsva.slack.com/archives/CT4GZBM8F/p1619547155126400) | Mismatch between WYSIWYG and preview server style.  Might deserve a followup issue. | 45m | Kevin Walsh, Steve Wirt, Neil Hastings, Kelson Adams |
|04/27| [Broken Link](https://dsva.slack.com/archives/C0MQ281DJ/p1619550105327500) | Article did not have an image, frontend templates assumed it did | 45m | Kevin Walsh, Steve Wirt, Neil Hastings, Adrian Rollett |
|04/28| [Question about uploading/referencing a PDF](https://dsva.slack.com/archives/CT4GZBM8F/p1619621959206100) | Trying to figure out if there's a pattern in place for this | 15m | Kevin Walsh |
|04/29| [Broken Link](https://dsva.slack.com/archives/CT4GZBM8F/p1619702097257000) [2](https://dsva.slack.com/archives/C01FMRGRAGY/p1619702615339100) | Archived page, learned that I shouldn't archive the page. | 45m | Steve Wirt |
|04/30| [Broken Link reported in CMS UI after fix](https://dsva.slack.com/archives/CDHBKAL9W/p1619797971302600) | Figured that it would be updated after the fixed revision was published, and it was. | 15m | |
|05/05| [Question about the Leadership list](https://dsva.slack.com/archives/CT4GZBM8F/p1620222315086300) | Figured that it would be updated after the fixed revision was published. | 20m | |
|05/05| [Question about adding a new video to Media List - Videos](https://dsva.slack.com/archives/CDHBKAL9W/p1620243692320500) | Already have an [issue](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/2763) open for this; it is blocked by the D9 upgrade | 20m | |
|05/06| [Failed Staging Deployment](https://dsva.slack.com/archives/CT4GZBM8F/p1620304589127500) | This failure occurs intermittently. I replayed the job and I think it will complete successfully. | 15m | |

**Comment by acrollet** [comment]
We're already getting close to a full day's effort and it's midweek in the first sprint. @ndouglas I've raised the estimate and dropped #4947 from the sprint, you can always grab it back if there ends up being time!

cc @mchelen-gov
